internal/kafka/command_consumer.go

```diff
@@ -0,0 +1,55 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v3/pkg/cmd"
+	"github.com/confluentinc/cli/v3/pkg/config"
+)
+
+type consumerCommand struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type consumerOut struct {
+	ConsumerGroupId string `human:"Consumer Group" serialized:"consumer_group_id"`
+	ConsumerId      string `human:"Consumer" serialized:"consumer"`
+	InstanceId      string `human:"Instance" serialized:"instance"`
+	ClientId        string `human:"Client" serialized:"client"`
```

Comment:
Should these also include "_id" in the serialized name, since the first one does?